### ISSUE TITLE
Sound on SteamOS

### ISSUE THREAD

**Issue by CrossplayGaming** [flathub/org.scummvm.ScummVM#43]
Hello,

I've been toying around with scummvm on steam deck (steamos) for a while now. I have gotten the neverhood working, but only when I use the windows version of scummvm via Proton. However, whenever I install the actual linux/steamos version, I can launch the game but there is no sound. Today, I tried wrath of the gods and it had the same issue. And for some reason the point and click functions of that game stop working in the wondows version via proton. So i'd like to finally get the actual linux version sound working so everything will behave in a more integrated way. Are there specific sound settings I have to tweak for the steamos version to get things moving? I've barely found anything on the topic online.

Thanks for any information about this.
Eric J.

**Comment by AsciiWolf** [comment]
Thanks for reporting the issue. The sound should work fine without tweaking any settings. I will try it on my Steam Deck and will let you know. Thanks!

**Comment by CrossplayGaming** [comment]
I have finally figured it out due to someone having a similar issue with retroarch! It turns out that by default, the steam deck was muting the Linux version of retroarch by default. I have no idea why as I had never opened the desktop mode sound settings up to this point. But once I did so with ScummVM opened in the background, sure enough it showed up as muted in the sound settings. Thanks for any time you took to dig into this regardless!
